docs/ML-Agents-Overview.md

```diff
@@ -684,6 +684,23 @@ three network architectures:
 The choice of the architecture depends on the visual complexity of the scene and
 the available computational resources.
 
+### Learning from Variable Length Observations using Attention
```

Comment:
- need to update the Toc above with a link to this section.
- any visuals we can add?

Reply:
I can try to make another comic book

Reply:
+1 for comic book.